source/file/Importer.js

```diff
@@ -52,8 +52,11 @@ class Importer {
 
   /** Determine paths' absolute points, given TopoJSON delta-encoded arcs */
   _getAbsolutePaths(geometry, allArcs, transform) {
+    // flatten MultiPolygon array so that we can map the arcs to get absolute paths
+    const arcs = geometry.type === 'Polygon' ?
```

Comment:
in the case the geometry type isn't Polygon or MultiPolygon should we just bail with an error? Not really a supported interaction regardless though.